lib/rack/mock.rb

```diff
@@ -257,7 +257,7 @@ def identify_cookie_attributes(cookie_filling)
       cookie_bits.each do |bit|
         if bit.include? '='
           cookie_attribute, attribute_value = bit.split('=')
```

Comment:
Would it make mores sense to use `split('=', 2)`?

Reply:
Yes, also works. Either way, the results of this function are discarded other than for these areas:
https://github.com/rack/rack/pull/1645/files#diff-4fc9bc1f7d91630f4f9f47fc6663f3f7L240-L245

It is probably incorrect to assume '=' is a divider.

A third option is not to process the first element of the `cookie_bits` array at all, since this is the value. So `cookie_bits[1..].each do |bit|`.

Fourth option, do both since the value could possibly also contain '='.

Reply:
Done both in f48f0b83